.github/workflows/run-tests.yml

```diff
@@ -116,7 +116,7 @@ jobs:
           echo QT_VERSION: ${QT_VERSION}
 
       - name: Login to Docker Hub
-        if: ${{ github.event_name == 'push' }}
+        if: ${{ github.event_name == 'push' && github.actor == 'qgis' }}
```

Comment:
This caused that the image hasn't been uploaded since then

Reply:
fixed in #60200

Reply:
@3nids thank you !

Reply:
thanks @3nids 
should the same adjustment also be applied for all the other changes from this PR?

Reply:
thanks Matthias, done in 0ebaa3986914d6e588c5251d26fb7120edbabd30